### PR TITLE
Implementation of handler, ver. 1.0

### PR DESCRIPTION
**Pull request by green1971weekend**
The branch add-task_handler include in itself an implementation of simple plans organizer. 
Organizer is capable of:
1. Add the new task or plan for a certain day.
2.Copy the all tasks from one day to another.
3.Delete the all saved tasks for a certain day.
4.Display the all appointed tasks for a certain day.
Also project provides protection against incorrect user input.

### CODE REVIEW

**Comment by MikhailMasny** [comment]
..handler, ver. 1.0 perceived better!

File: Straltsou.TMS0209.sln [hunk 1]
```diff
@@ -3,24 +3,30 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29911.84
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleCalculator", "SimpleCalculator\SimpleCalculator.csproj", "{AE4D92B3-F3D0-4803-9FF2-72506EE4A733}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HelloWorldProj", "src\HelloWorld\HelloWorldProj.csproj", "{850F69B5-2425-4CE3-BEE6-40E08BB08CB3}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleCalculator", "src\SimpleCalculator\SimpleCalculator.csproj", "{F8777F82-9547-44A0-A982-9681D04FCFD4}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TaskHandler", "..\..\TMS Test\Tasks Draft\Example1\TaskHandler.csproj", "{811E24BC-480E-4383-A5CF-D47F7BB16720}"
```
Comment (MikhailMasny) [inline Straltsou.TMS0209.sln:10]: Bad path to project. Change it to: "TaskHandler", "src\TaskHandler\TaskHandler.csproj"

File: src/TaskHandler/Program.cs [hunk 2]
```diff
@@ -0,0 +1,286 @@
+﻿using System;
```
Comment (MikhailMasny) [inline src/TaskHandler/Program.cs:1]: You have some bugs.. But all seems good 👍

**Review by MikhailMasny — APPROVED** [review]
Fix it over time, and so everything is cool.

**Comment by green1971weekend** [comment]
Thank you for advise.

**Pull request merged**